### ISSUE TITLE
Rule Change: lines-around-comment

### ISSUE THREAD

**Issue by MuTsunTsai** [eslint/eslint#16750]
### What rule do you want to change?

lines-around-comment

### What change to do you want to make?

Generate more warnings

### How do you think the change should be implemented?

A new option

### Example code

```js
/**
 * Some block comments
 */

function func() { }
```

### What does the rule currently do for this code?

Doesn't do anything. Currently the rule only enforces new lines around comments, not the other way.

### What will the rule do after it's changed?

I would like it to have an new value besides `true` and `false` (such as `"never"`) that enforces the opposite, that is, there CANNOT be new lines around comments in certain cases. In the example above, I want to make sure that there's no new line between the block comments and the function, so I would do something like `"afterBlockComment": "never"`.

### Participation

- [ ] I am willing to submit a pull request to implement this change.

**Comment by mdjermanovic** [comment]
Hi @MuTsunTsai, thanks for the issue!

Per our rules policies, [stylistic rules are frozen](https://github.com/eslint/eslint#stylistic-rule-updates), so we are not adding new options to the `lines-around-comment` rule. You can always copy the existing rule into a custom rule and make it behave the way you want.
